src/backend/src/services/projects.services.ts

```diff
@@ -123,7 +140,177 @@ export default class ProjectsService {
           }
         }
       },
-      include: { project: true, changes: true }
+      include: {
+        project: { include: { goals: true, features: true, otherConstraints: true } },
+        changes: true,
+        links: true
+      }
+    });
+    const { wbsElementId } = createdWbsElement;
+
+    // const createdProject = createdWbsElement.project;
+    const createdProject = await prisma.project.findUnique({
+      where: { wbsElementId },
+      include: { goals: true, features: true, otherConstraints: true }
+    });
+    // if it doesn't exist we error
+    if (!createdProject) throw new NotFoundException('Project', wbsElementId);
+
```

Comment:
Rather than just having the project come from `createdWbsElement`, I had to get it from Prisma because it's not included in `createdWbsElement` despite creating and including it in the create wbs element function

Reply:
i noticed that as well